distributed/worker.py

```diff
@@ -945,6 +945,7 @@ async def get_metrics(self) -> dict:
                 "memory": spilled_memory,
                 "disk": spilled_disk,
             },
+            event_loop_interval=self._tick_interval_observed,
```

Comment:
I don't see this being used anywhere.

I'm increasingly concerned about the heartbeat since we're putting more and more on it. The metrics in particular are even logged in a deque as an event on scheduler side. Most worker event logs are full of these heartbeats but I haven't seen this being used anywhere. I don't think we should submit anything and most definitely not store anything if we don't have a usage for it.

This single float is not a big deal, of course. Just wondering.

Reply:
It ends up as a metric in the WorkerState.  That metric gets pulled by the dashboard plot.

I'm totally open to dropping heartbeats from the eventing system